content/en/metrics/otlp.md

```diff
@@ -121,14 +121,13 @@ You may add all resource attributes as tags by using the `resource_attributes_as
 OpenTelemetry defines certain semantic conventions related to host names. If an OTLP payload has a known hostname attribute, Datadog honors these conventions and tries to use its value as a hostname. The semantic conventions are considered in the following order:
 
 1. `datadog.host.name`, a Datadog-specific hostname convention
-2. `k8s.node.name`, the Kubernetes node name
-3. Cloud provider-specific conventions, based on the `cloud.provider` semantic convention
-4. `host.id`, the unique host ID
-5. `host.name`, the system hostname
-6. `container.id`, the container ID
+1. Cloud provider-specific conventions, based on the `cloud.provider` semantic convention
+1. Kubernetes-specific conventions from the `k8s.node.name` and `k8s.cluster.name` semantic conventions.
```

Comment:
None of the other points in this list end with a `.`, so maybe this one shouldn't as well?

Reply:
Makes sense, removed